packages/dnb-eufemia/src/extensions/forms/hooks/useFieldProps.ts

```diff
@@ -343,10 +345,10 @@ export default function useFieldProps<Value, EmptyValue, Props>(
     dataContextError
   )
 
-  const onChangeValidatorRef = useRef(validator)
+  const onChangeValidatorRef = useRef(onChangeValidator)
   useUpdateEffect(() => {
-    onChangeValidatorRef.current = validator
-  }, [validator])
+    onChangeValidatorRef.current = onChangeValidator
+  }, [onChangeValidator]) // Tobias, will this still work? now that we do onChangeValidator = validator?
```

Comment:
```suggestion
  }, [onChangeValidator])
```

Will this still work with the `validator` property, now that we use validator as default value for `onChangeValidator` (`onChangeValidator = validator`)? Or will we have to listen for changes to `validator` as well? 🤔 
Or do we have to listen for changes in

Reply:
Yes, this will still work, we even have tests for this where it re-renders, `should return error when validator callback returns error`.